FourParameterCorrelationModel.cpp

```diff
@@ -22,20 +22,24 @@
 //     20-Dec-2012   JPK      Fixed bug in constructor.
 //     18-Feb-2013   JPK      Removed getNumCorrelationParameterGroups,
 //                            now provided on base class.
-//     03-Sep-2021   SCM      Removed IRIX support.
```

Comment:
You seem to have removed this history entry (and likely the associated changes).